src/shared/components/variant-alleles-sequences/VariantAllelesSequences.tsx

```diff
@@ -0,0 +1,116 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+
+import ShowHide from 'src/shared/components/show-hide/ShowHide';
+
+import styles from './VariantAlleleSequences.scss';
+
+const COMPACT_MAX_LENGTH = 18;
+
+type Props = {
+  isExpandable?: boolean;
+  alleles: {
```

Comment:
Isn't it just one allele's details that gets passed on?

Reply:
There's a `[]` on line 30 🙂

Reply:
Yes I missed that one. I need better glasses 😅